ros_cross_compile/ros_cross_compile.py

```diff
@@ -128,21 +134,49 @@ def parse_args(args: List[str]) -> argparse.Namespace:
         default=[],
         nargs='+',
         help='Skip specified rosdep keys when collecting dependencies for the workspace.')
+    parser.add_argument(
+        '--print-metrics',
+        action='store_true',
+        required=False,
+        help='Print the collected metrics to its own logging level, METRIC. This option'
+             'is provided to give the user flexibility when working with automated solutions'
+             'and may not want stdout output.')
+    parser.add_argument(
+        '--metric-file-path',
+        required=False,
+        type=str,
```

Comment:
Wondering if we could set this to type=Path so that we get directly the "right type"?